components/base/FilterTabs.tsx

```diff
@@ -69,11 +75,13 @@ export default function FilterTabs({ tabs, filter, setFilter }: Props) {
       <TabsContainerStyle>
         {formattedTabs.map((tab) => (
           <TabStyle
+            tabIndex={0}
             className={classnames("tab", {
               "is-active":
                 JSON.stringify(formattedFilter) === JSON.stringify(tab),
             })}
             key={tab.value}
+            onFocus={() => setFilter(tab)}
```

Comment:
Instead of having different Hover/Filter styles, I ended up changing the Filter on focus as well. In other words, focus will render filtered content based on tab change.

Reply:
Hmmm, I really liked this when you were describing it, but now that I try it out, I'm not sure this is the best solution. If someone is navigating by keyboard only (as would be the case with a screen reader), they would be unable to get to the cards without inadvertently changing the filter to the last option

Reply:
Oh wow! Great catch! Can't believe I missed that. So the best solution is probably something similar to how keyboard selection works on the `<select>` component. Thanks Connor!

Reply:
This should fix it - https://github.com/Dwolla/dev-portal/pull/106/commits/51f1819764827bc7b1a31862d3b378b6eb2f8f33